spk/bazarr/Makefile

```diff
@@ -0,0 +1,57 @@
+SPK_NAME = bazarr
+
+include ../../cross/bazarr/bazarr.mk
+
+SPK_VERS = $(BAZARR_VERSION)
+SPK_REV = 2
+
+SPK_ICON = src/bazarr.png
+
+REQUIRED_DSM = 6.1
+
+BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel
+
+BETA = 1
+
+DEPENDS = cross/busybox
+DEPENDS += cross/libxml2 
+DEPENDS += cross/libxslt 
```

Comment:
I would replace these dependencies by `cross/lxml` and update it if needed - and I guess it is then possible to remove `WHEELS` variable.

Reply:
Indeed, and it removed the requirement for the headers files !